ui/app/services/flags.ts

```diff
@@ -86,4 +88,21 @@ export default class flagsService extends Service {
       this.secretsSyncActivatePath.get('canUpdate') !== false
     );
   }
+
+  get showSecretsSync() {
+    const isHvdManaged = this.isHvdManaged;
+    const onLicense = this.version.hasSecretsSync;
+    const isEnterprise = this.version.isEnterprise;
+    const isActivated = this.secretsSyncIsActivated;
+
+    if (!isEnterprise) return false;
+    if (isHvdManaged) return true;
```

Comment:
Do we want to rearrange this order here and have `isHvdManaged` first? Since HVD clusters aren't enterprise...right? So this getter would return early for HVD situations

Reply:
they sure are 🙂